### DISCUSSION TITLE
Remove a song in the background with the vocals

### DISCUSSION THREAD

**Discussion by taz1004** [Anjok07/ultimatevocalremovergui#1301] [Q&A]
I have a people talking at a cafe and there's a song in the background with vocals.  I'm trying to remove the song (vocal and instrumental) and only keep the conversation.  Is this possible?

The song is a popular song and I have it as a separate file.  Is there any way to compare to the song only and remove?

**Comment by vdanelia** [comment]
I have the same question, please help us
How to remove a song (with music) from the background of the video if i choose to remove a vocal, it removes the video speech too.. 
Thank you
